ingestion/src/metadata/ingestion/ometa/ometa_api.py

```diff
@@ -571,3 +573,55 @@ def close(self):
             None
         """
         self.client.close()
+
+    def _get_formmated_table_name(self, table_name):
+        return table_name.replace("[", "").replace("]", "")
+
+    def _create_lineage(self, from_table, to_table, service_name):
+        try:
+            from_fqdn = (
+                f"{service_name}.{self._get_formmated_table_name(str(from_table))}"
+            )
+            from_entity: Table = self.get_by_name(entity=Table, fqdn=from_fqdn)
+
+            to_fqdn = f"{service_name}.{self._get_formmated_table_name(str(to_table))}"
+            to_entity: Table = self.get_by_name(entity=Table, fqdn=to_fqdn)
+            if not from_entity or not to_entity:
+                return None
+
+            lineage = AddLineageRequest(
+                edge=EntitiesEdge(
+                    fromEntity=EntityReference(
+                        id=from_entity.id.__root__,
+                        type="table",
+                    ),
+                    toEntity=EntityReference(
+                        id=to_entity.id.__root__,
+                        type="table",
+                    ),
+                )
+            )
+
+            created_lineage = self.add_lineage(lineage)
+            logger.info(f"Successfully added Lineage {created_lineage}")
+
+        except Exception as err:
+            logger.debug(traceback.print_exc())
+            logger.error(err)
+
+    def ingest_lineage(self, query, service_name):
+        from sqllineage.runner import LineageRunner
```

Comment:
Same as above. Maybe move that to a utility module and add proper naming, hints and docstrings. This method does not generally ingest_lineage, it is a wrapper on some SQL lineage logic for specific cases.